setup.py

```diff
@@ -7,10 +7,14 @@
 """
 import codecs
 import os
+import sys
 
-from nodeenv import nodeenv_version
 from setuptools import setup
 
+sys.path.insert(0, '.')
+
+from nodeenv import nodeenv_version
```

Comment:
this relies on `packaging` at *build* time -- which is probably not intended

Reply:
That’s a consequence of importing the to-be-built package at build time, which is a behavior not introduced by this PR.

There is no harm in in keeping it that way (`packaging` is small and, unlike `pkg_resources`, very fast to import), but if @ekalinin wants to, he can of course move the `packaging` imports into the functions where `packaging.version.parse()` is called. I don’t really see why that’s necessary though.